scripts/app-config-replace.js

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env node
+
+const program = require('commander');
+require('dotenv').config({ path: process.env.ENV_FILE });
+const fs = require('fs');
+
+const API_HOST = process.env.API_HOST || 'api';
+const OAUTH_HOST = process.env.OAUTH_HOST || 'keycloak';
+
+const options = {
+  apiHost: {
+    flags: '-a, --api-host',
+    description: "set apiHost's and ecmHost's value with API_HOST",
+    set: appConfig => {
+      appConfig.ecmHost = API_HOST;
+      appConfig.aosHost = API_HOST + '/alfresco/aos';
+      appConfig.baseShareUrl = API_HOST + '/#/preview/s';
```

Comment:
this should be E2E_HOST

Reply:
nope this is the backend the e2e_host is where the FE is

Reply:
yes, and the baseShareUrl is the one used for public shared files, and should be the URL of the content-app / digital workspace, not the URL of the backend.